src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -70,10 +70,12 @@ private static string GetSchema(ResourceScope targetScope)
         private EmitterContext Context => ExpressionBuilder.Context;
         private ExpressionBuilder ExpressionBuilder { get; }
         private ImportClosureInfo ImportClosureInfo { get; }
+        private SemanticModel Model { get; }
         private ImmutableDictionary<string, DeclaredTypeExpression> declaredTypesByName;
 
         public TemplateWriter(SemanticModel semanticModel)
         {
+            Model = semanticModel;
```

Comment:
This is already accessed in this class with `Context.SemanticModel`. For consistency, we should pick one or the other, and udpate all usages.